src/SFA.DAS.EmployerAccounts/Data/EmployerAgreementRepository.cs

```diff
@@ -151,5 +152,19 @@ public async Task<AccountLegalEntityModel> GetAccountLegalEntity(long accountLeg
 
             return result.SingleOrDefault();
         }
+
+        public async Task<IEnumerable<EmployerAgreement>> GetAccountAgreements(long accountId)
```

Comment:
We seem to have various methods to access the DB should we adopt a consistent approach?